packages/blocks/src/page-block/keyborad/keyboard-manager.ts

```diff
@@ -195,10 +196,8 @@ export class Keyboard {
   ) {
     const current = selections[0];
     const first = this._page.getBlockById(current.blockId);
-    const firstElement = this.host.root.viewStore.viewFromPath(
-      'block',
-      current.path
-    );
+    const firstElement = this.host.root.viewStore.getViewByPath(current.path)
```

Comment:
Can be replaced with `const firstElement = this.host.root.viewStore.viewFromPath('block', current.path)`.